Build failure: no member named 'microhardManager' in 'QGCToolbox'

I'm trying to build QGC (haven't succeeded yet) and getting this error:
```
In file included from /home/phma/src/qgroundcontrol/src/Microhard/MicrohardSettings.cc:13:
In file included from /home/phma/src/qgroundcontrol/src/QGCApplication.h:27:
In file included from /home/phma/src/qgroundcontrol/src/FirmwarePlugin/FirmwarePluginManager.h:14:
/home/phma/src/qgroundcontrol/src/FirmwarePlugin/FirmwarePlugin.h:334:44: warning: unused parameter 'vehicle' [-Wunused-parameter]
    virtual QString getHobbsMeter(Vehicle* vehicle) { return "Not Supported"; }
                                           ^
/home/phma/src/qgroundcontrol/src/Microhard/MicrohardSettings.cc:83:53: error: no member named 'microhardManager' in 'QGCToolbox'
        std::string userName = qgcApp()->toolbox()->microhardManager()->configUserName().toStdString() + "\n";
                               ~~~~~~~~~~~~~~~~~~~  ^
/home/phma/src/qgroundcontrol/src/Microhard/MicrohardSettings.cc:86:48: error: no member named 'microhardManager' in 'QGCToolbox'
        std::string pwd = qgcApp()->toolbox()->microhardManager()->configPassword().toStdString() + "\n";
                          ~~~~~~~~~~~~~~~~~~~  ^
/home/phma/src/qgroundcontrol/src/Microhard/MicrohardSettings.cc:92:34: error: no member named 'microhardManager' in 'QGCToolbox'
            qgcApp()->toolbox()->microhardManager()->setEncryptionKey();
            ~~~~~~~~~~~~~~~~~~~  ^
1 warning and 3 errors generated.
ninja: build stopped: subcommand failed.
```
## System Information
When posting bug reports, include the following information
- Operating System: Ubuntu Focal
- QGC Version: Just pulled from git
- QGC build: not successfully built yet
- Flight Controller: n/a
- Autopilot (with version): n/a
- CMake: 3.16.3
- Ninja: 1.10.0
- Clang: 10.0.0-4ubuntu1

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -107,9 +107,10 @@ endif()
 if (GST_FOUND)
     add_definitions(
         -DQGC_GST_STREAMING
-        -DQGC_GST_TAISYNC_ENABLED
-        -DQGC_GST_MICROHARD_ENABLED
     )
+
+    option(QGC_GST_MICROHARD_ENABLED "Enable microhard" OFF)
+    option(QGC_GST_TAISYNC_ENABLED "Enable taisyng" OFF)
 endif()
 
 add_definitions(

